payloads/external/sortbootorder/Makefile

```diff
@@ -39,7 +39,7 @@ checkout: fetch
 
 build: checkout
 	echo "    MAKE       sortbootorder"
-	$(MAKE) -C sortbootorder VERSION=$(TAG-y) APU1=$(TARGET_APU1)
+	$(MAKE) -C sortbootorder VERSION=$(TAG-y) APU1=$(TARGET_APU1) PXE=$(PXE)
```

Comment:
I would handle it entirely in sortbootorder. You can retrieve the board name in the payload and then act accordingly there.

Reply:
Done, please see https://github.com/pcengines/sortbootorder/pull/81